tests/test_query.py

```diff
@@ -41,19 +46,45 @@ def test_query_should_add_language(self):
     def test_query_should_add_start_time(self):
         q = self._client.query()
         q.pattern = "spotify"
-        q.start_time = datetime.datetime(2012, 12, 28, 9, 1, 22)
+        q.start_time = self.datetime_with_timezone(datetime(2012, 12, 28, 9, 1, 22), "UTC")
         self.assertEqual(q.request_parameters()['ts'], "2012-12-28 09:01:22")
 
+    def test_query_using_start_time_without_timezone(self):
+        q = self._client.query()
+        q.pattern = "spotify"
+        q.start_time = datetime(2012, 12, 28, 9, 1, 22)
+        with self.assertRaises(twingly_search.TwinglyQueryException):
```

Comment:
It's different for Ruby and Node.js, because they default to your local timezone, so we always have that information. We haven't done PHP yet, but PHP will yell at you if you don't configure your timezone or explicitly pick one when creating the date object (as we do in our examples).